libqtile/layout/tile.py

```diff
@@ -47,8 +47,10 @@ class Tile(_SimpleLayoutBase):
     defaults = [
         ("border_focus", "#0000ff", "Border colour(s) for the focused window."),
         ("border_normal", "#000000", "Border colour(s) for un-focused windows."),
+        ("border_on_single", 1, "Draw border if there is only one window."),
         ("border_width", 1, "Border width."),
         ("margin", 0, "Margin of the layout (int or list of ints [N E S W])"),
+        ("margin_on_single", 1, "Draw margin if there is only one window."),
```

Comment:
```suggestion
        ("margin_on_single", True, "Whether to draw margin if there is only one window."),
```

And a similar change for `border_on_single`.

This will make it more clear that the option takes `True` or `False` rather than an `int`, which may make people think that the option configures the border size for when there is one window.